payments/invoice/other-features.md

```diff
@@ -19,108 +19,677 @@ sidebar:
+|          | └➔&nbsp;`pickUpAddress`                  | `object`      | If shipIndicator set to 4, then prefil this.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+|          | └─➔&nbsp;`name`                          | `string`      | If shipIndicator set to 4, then prefil this.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+|          | └─➔&nbsp;`streetAddress`                 | `string`      | If shipIndicator set to 4, then prefil this.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+|          | └─➔&nbsp;`coAddress`                     | `string`      | If shipIndicator set to 4, then prefil this.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+|          | └─➔&nbsp;`city`                          | `string`      | If shipIndicator set to 4, then pr
```

Comment:
replace "input creditcard information" with the correct value corresponding to Invoice Payments